internal/expr/expr_test.go

```diff
@@ -411,6 +414,15 @@ comment */
 string
 of three lines' AND id = $Person.*`,
 		err: `cannot parse expression: line 3, column 26: asterisk not allowed in input expression "$Person.*"`,
+	}, {
+		query: "SELECT &S[:] FROM t",
+		err:   `cannot parse expression: cannot use slice type "S" in output expression: &S[:]`,
```

Comment:
This is an idea, but do you think we could do without mentioning the type `S` in the message, I find the phrase "slice type" a little bit weird, what do you think of the following?
```
cannot parse expression: cannot use slice in output expression: &S[:]
```

Reply:
Sounds good, changed.